snapcraft.yaml

```diff
@@ -128,7 +123,7 @@ parts:
   probert:
     plugin: python
     build-packages: [python-setuptools, libnl-3-dev, libnl-genl-3-dev, libnl-route-3-dev]
-    stage-packages: [libc6, libnl-3-dev, libnl-genl-3-dev, libnl-route-3-dev]
+    stage-packages: [libnl-3-dev, libnl-genl-3-dev, libnl-route-3-dev]
```

Comment:
We don't need -dev packages staged, just the runtime libs, no? And also, because probert is in the core18 snap, we don't need to stage any of these libnl* libs anyway.

Reply:
Hm good point, lets try that.